tests/utils.go

```diff
@@ -3495,7 +3495,7 @@ func CreateNFSTargetPOD(os string) (nfsTargetIP string) {
 	getStatus := func() k8sv1.PodPhase {
 		pod, err := virtClient.CoreV1().Pods(NamespaceTestDefault).Get(pod.Name, metav1.GetOptions{})
 		Expect(err).ToNot(HaveOccurred())
-		nfsTargetIP = pod.Status.PodIP
+		podIPs = pod.Status.PodIPs
 		return pod.Status.Phase
```

Comment:
This could lead to unexpected behaviour, having specified return value in the function signature are returning another variable
You should remove the name of the returned value from the function signature and change this line to `return pod.Status.PodIPs`